blocks/columns/luxury-collection-template.css

```diff
@@ -0,0 +1,74 @@
+.luxury-collection .columns.block {
+    max-width: var(--normal-page-width);
+}
+
+.luxury-collection .columns.block h2 {
+    font: normal normal 300 50px/56px var(--font-family-gotham);
+    letter-spacing: 0;
+    margin-bottom: 25px;
+}
+
+.luxury-collection .columns.block h3 {
+    font: normal normal 700 16px/23px var(--font-family-gotham);
+    letter-spacing: 2.08px;
+    margin-bottom: 24px;
+}
+
+.luxury-collection .columns.block p{
```

Comment:
I don't know if we want to have the Lux experience be unique to it, or if we can just use the existing defintions, and just change the color and maybe the fonts.

Having a fully unique experience for lux seems over the top and too much effort to maintain in the long run?